examples/multiple-contracts/Cargo.toml

```
@@ -1,14 +1,16 @@
 [package]
 name = "multiple-contracts"
-authors = ["Cardinal", "Aleph Zero Foundation"]
+authors = ["Use Ink <ink@use.ink>"]
```

```suggestion
authors = ["Cardinal", "Aleph Zero Foundation", "Use Ink <ink@use.ink>"]
```
Let's keep the credit for the original authors 🙂